Form-Controls/index.html

```diff
@@ -9,19 +9,53 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
   <header>
-    <h1>Product Pick</h1>
+    <h1>T-shirt order form</h1>
+    <h2 id="heading">choose your t-shirt here!</h2>
```

Comment:
IDs must be unique. Is #heading therefore a good ID for this heading? Would another developer understand that you mean this heading out of all the headings on this page?